bin/github.js

```diff
@@ -19,24 +16,32 @@ yargs
 	 * @see https://github.com/yargs/yargs/blob/master/docs/advanced.md#commanddirdirectory-opts
 	 */
 	.commandDir(yargsCommandsDirectoryPath)
+	/**
+	 * Set a usage message and description.
+	 */
+	.usage('$0 <command> <subcommand> [arguments]', 'Work with GitHub from the command line.')
```

Comment:
Todo: Change ['arguments'] to ['options'] (or change all instances of '[options]' to '[arguments]'